provision-sm-md-lg/README.md

```diff
@@ -0,0 +1,81 @@
+# Provision VM Service Small Medium Large Example
+
+This example shows creating a provisioning dialog that modifies provisioning parameters
+
+## Architecture
+
+- `provision-vm-small-med-large.asl`: This workflow is run for every vm provisioned and is responsible for modifying the vm size. This will be used as a ManageIQ Service's Provisioning Entry Point.
+
+## Setup within ManageIQ
+
+### Set up the workflow repository
+
+#### Add this repository
+
+1. Automation -> Embedded Workflows -> Repositories.
+2. Configuration -> Add new Repository
+3. `Name = Example Workflows`
+4. `URL = https://github.com/ManageIQ/workflows-examples`
```

Comment:
Step3 and 4 can be probably combined in one step.

Provide the name and url for the repository. An example name and url can resemble the following:
`Name: Example Workflows`
URL: https://github.com/ManageIQ/workflows-examples`

Reply:
I worked on this one. Not sure if I got your intention here.